### ISSUE TITLE
Spectacle docs: website doesn't load when using https://commerce.nearform.com/open-source/spectacle

### ISSUE THREAD

**Issue by Burnett2k** [FormidableLabs/spectacle#1326]
if you add a trailing slash it does work: 

https://commerce.nearform.com/open-source/spectacle/ (works)
vs
https://commerce.nearform.com/open-source/spectacle (doesn't work)

**Comment by Burnett2k** [comment]
www links also don't seem to work (i.e. https://www.commerce.nearform.com/open-source/spectacle/docs vs https://commerce.nearform.com/open-source/spectacle/docs)

**Comment by ryan-roemer** [comment]
Yeah, I'm guessing we just need to audit and replace any `www.commerce.nearform.com` links
